packages/spartan/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "spartan": {
       "kind": "git",
       "git": "https://github.com/dcos/spartan.git",
-      "ref": "0b5243b8032bf475fb2c62ec4f92142d701f35ad",
+      "ref": "f7b2311c71fa833c182a13781af00a387632fb1b",
```

Comment:
@amitaekbote Can you update this train with the new ref from #1536 ?
Either remove the old commits and re-add the new ones, or just change the ref to `dcos/1.9.1` if that's all that changed.